apps/ewallet_db/lib/ewallet_db/transaction_request.ex

```diff
@@ -301,7 +296,7 @@ defmodule EWalletDB.TransactionRequest do
       set: [
         status: @expired,
         expired_at: NaiveDateTime.utc_now(),
-        expiration_reason: "expired_transaction_request"
+        expiration_reason: @expired_transaction_request
```

Comment:
Better use `TransactionRequest.expired_transaction_request()` consistently so we know we use the same value throughout the system.